### PR TITLE
GNST getnx + peers

### PR DESCRIPTION
**Pull request by Genstamindband**

### CODE REVIEW

File: quark/gentxs/gentx.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"GNST","identity":"","website":"https://github.com/Genstamindband","security_contact":"","details":""},"commission":{"rate":"0.100000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1wggg4ge3lgc5zpa3njk9eyxjkduatsedf63vr9","validator_address":"neutronvaloper1wggg4ge3lgc5zpa3njk9eyxjkduatsedn8g49p","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"J1EtE+muraNaRHs+9rTqaA3w/Jf/xqTVxO6u4PcXknc="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"22b66bec1ca8879fe517143884ea7451b1d08d79@38.242.148.96:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A4DZYDUsYX2L06HHIvZthYn0DJEQz92GsH4YKPOzSyw/"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["gkT7tAOTzVLT9mE5OJXVpnkXlSoXMFV1YFtmE9MAXpgSm4e/gnLnb3wgrXWH5Wz81wHPeg9mKarKaJJ78WUwkw=="]}
```
Comment (albertandrejev) [inline quark/gentxs/gentx.json:1]: Please update gentx file name to something unique. You can rename it to something like: `gentx-gnst.json` for example.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/gentx.json (hunk 1 above)
Comment (Genstamindband) [reply quark/gentxs/gentx.json]: Done!

File: quark/gentxs/README.md [hunk 2]
```diff
@@ -1 +1,7 @@
-## Please put your gentx into this directory
\ No newline at end of file
+Moniker: GNST
```
Comment (albertandrejev) [inline quark/gentxs/README.md:1]: You do not need to put anything into this file, PR comment is enough. Please remove changes from this file

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/README.md (hunk 2 above)
Comment (Genstamindband) [reply quark/gentxs/README.md]: Done

**Comment by Genstamindband** [comment]
Webpage: https://twitter.com/genstamindband

 Discord handle: Genstamindband#6713, FlakySizzler#9185, hedji#5037, DimaRCR7#0690, Dr_Atavov_PRO#0484

 neutronvaloper1wggg4ge3lgc5zpa3njk9eyxjkduatsedn8g49p

**Pull request closed without merging**